lib/tasks/acls.rake

```diff
@@ -71,6 +71,29 @@ namespace :acls do
     end
   end
 
+  namespace :subscriptions do
+    desc 'Creates a group and grants permission for managing subscriptions'
+    task :full_acls, [:username] => :environment do |_task, args|
+      # Create the group
+      Rake::Task['acls:groups:create'].invoke('Subscription Management', 'Group Created for Subscription Management', args.username, 'MMT_2')
```

Comment:
if we need to be able to ingest subscriptions with this, you might need to also add `testuser` to the group, as the token we pass in development and test is for that user